CHANGELOG.md

```diff
@@ -1,3 +1,41 @@
+CHANGES IN VERSION 1.4
+======================================
+
+#### General
+- fix for broken `./reset.sh --dev` due broken UICatalog build package in sample-code submodule
+- fix for issues with cookie encoding
+- updated sample code
+- updated documentation
+- updated test suite
+
+#### iOS
+- Support for iOS 8.3
+- fix for issues relating to finding xcode folder
+- new sever flag `--instruments <path>` to specify custom path to instruments commandLine tool
+- fix for getOrientation
+- fix for iOS crash log retrieval
+
+#### Android
+- fix for killing chromedriver on windows
+- fix for parsing java version correctly
+- support for searching elements by id without passing package name
+- capability object to have deviceName and platformVersion
```

Comment:
- requesting capabilities from the server now returns the correct deviceName and platformVersion on Android